src/group_theory/subgroup/basic.lean

```diff
@@ -1154,6 +1154,81 @@ ext $ λ x, by simp [mem_pi]
 (eq_bot_iff_forall _).mpr $ λ p hp,
 by { simp only [mem_pi, mem_bot] at *, ext j, exact hp j trivial, }
 
+@[to_additive]
+lemma le_pi_iff {I : set η} {H : Π i, subgroup (f i)} {J : subgroup (Π i, f i)} :
+  J ≤ pi I H ↔ (∀ i : η , i ∈ I → map (pi.eval_monoid_hom f i) J ≤ H i) :=
```

Comment:
You should be able to abbreviate this to:
```suggestion
  J ≤ pi I H ↔ (∀ i ∈ I, map (pi.eval_monoid_hom f i) J ≤ H i) :=
```

Reply:
Ah, I remember why this breaks stuff: This syntax introduces a hypothesis called `H`, which shadows `H`.